data/human/campaign events.txt

```diff
@@ -1753,7 +1753,7 @@ event "fw northern expansion"
 
 event "fw prison on Clink"
 	planet "Clink"
-		add description `Recently, the Free Worlds have converted Clink into a prison colony for Navy officers and support staff who were captured in the battle for the Rastaban system.`
+		add description `Recently, the Free Worlds has converted Clink into a prison colony for Navy officers and support staff who were captured in the battle for the Rastaban system.`
```

Comment:
`have` is actually more widely used in the game, and I think it describes the federate aspect of the FW better than `has` would.

```suggestion
		add description `Recently, the Free Worlds have converted Clink into a prison colony for Navy officers and support staff who were captured in the battle for the Rastaban system.`
```